samza-core/src/main/java/org/apache/samza/config/DefaultChooserConfig.java

```diff
@@ -30,15 +30,17 @@
  * A convenience class for fetching configs related to the {@link org.apache.samza.system.chooser.DefaultChooser}
  */
 public class DefaultChooserConfig extends MapConfig {
-  public static final String BOOTSTRAP = StreamConfig.STREAM_PREFIX() + "samza.bootstrap";
-  public static final String PRIORITY = StreamConfig.STREAM_PREFIX() + "samza.priority";
-  public static final String BATCH_SIZE = "task.consumer.batch.size";
+  private static final String BOOTSTRAP = StreamConfig.SAMZA_PROPERTY() + "bootstrap";
```

Comment:
I thought about this and have no strong feelings about it. 

This would basically mean deleting this class, as it provides no other value.

Preference?